src/components/FormFooter.tsx

```diff
@@ -0,0 +1,80 @@
+import classNames from 'classnames';
+import React from 'react';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  todos: Todo[],
+  filter: string,
+  setFilter: (filter: string) => void;
+};
+
+export const FormFooter: React.FC<Props> = ({ todos, filter, setFilter }) => {
+  return (
+    <footer className="todoapp__footer">
+      <span className="todo-count">
+        {`${(todos.filter(todo => !todo.completed)).length} items left`}
+      </span>
+
+      <nav className="filter">
+        <a
+          href="#/"
+          className={classNames(
+            'filter__link',
+            { selected: filter === 'all' },
+          )}
+          onClick={() => {
+            setFilter('all');
+          }}
+        >
+          All
+        </a>
+
+        <a
+          href="#/"
+          className={classNames(
+            'filter__link',
+            { selected: filter === 'active' },
+          )}
+          onClick={() => {
+            setFilter('active');
+          }}
+        >
+          Active
+        </a>
+
+        <a
+          href="#/"
+          className={classNames(
+            'filter__link',
+            { selected: filter === 'completed' },
+          )}
+          onClick={() => {
+            setFilter('completed');
+          }}
+        >
+          Completed
+        </a>
+      </nav>
+
+      {/* don't show this button if there are no completed todos */}
+      {(todos.filter(todo => todo.completed)).length > 0
```

Comment:
make a separate variables for the conditions like that. It is not easy to read this way